frontend/packages/process-editor/src/components/ConfigPanel/ConfigPanel.test.tsx

```diff
@@ -37,124 +36,89 @@ const mockBpmnContextValue: BpmnContextProps = {
-    ).toBeInTheDocument();
-  });
-
-  it('should display the details about the selected task when a "data" task is selected', () => {
-    render();
-
-    expect(
-      screen.getByRole('heading', {
-        name: textMock('process_editor.configuration_panel_data_task'),
-        level: 2,
-      }),
-    ).toBeInTheDocument();
-
-    expect(screen.getByText(mockBpmnDetails.id)).toBeInTheDocument();
-    expect(screen.getByText(mockBpmnDetails.name)).toBeInTheDocument();
-  });
-
-  it('should display the details about the selected task when a "confirmation" task is selected', () => {
-    render({ bpmnDetails: { ...mockBpmnDetails, taskType: 'confirmation' } });
+    (shouldDisplayFeature as jest.Mock).mockReturnValue(true);
+    renderConfigPanel({ bpmnDetails: { ...mockBpmnDetails, type: BpmnTypeEnum.EndEvent } });
 
     expect(
-      screen.getByRole('heading', {
-        name: textMock('process_editor.configuration_panel_confirmation_task'),
-        level: 2,
-      }),
-    ).toBeInTheDocument();
-
-    expect(screen.getByText(mockBpmnDetails.id)).toBeInTheDocument();
-    expect(screen.getByText(mockBpmnDetails.name)).toBeInTheDocument();
-  });
-
-  it('should display the details about the selected task when a "feedback" task is selected', () => {
-    render({ bpmnDetails: { ...mockBpmnDetails, taskType: 'feedback' } });
-
-    expect(
-      screen.getByRole('heading', {
-        name: textMock('process_editor.configuration_panel_feedback_task'),
-        level: 2,
-      }),
+      screen.getByText(textMock('process_editor.configuration_panel_end_event')),
     ).toBeInTheDocument();
-
-    expect(screen.getByText(mockBpmnDetails.id)).toBeInTheDocument();
-    expect(screen.getByText(mockBpmnDetails.name)).toBeInTheDocument();
   });
 
-  it('should display the details about the selected task when a "signing" task is selected', () => {
-    render({ bpmnDetails: { ...mockBpmnDetails, taskType: 'signing' } });
-
-    expect(
-      screen.getByRole('heading', {
-        name: textMock('process_editor.configuration_panel_signing_task'),
-        level: 2,
-      }),
-    ).toBeInTheDocument();
-
-    expect(screen.getByText(mockBpmnDetails.id)).toBeInTheDocument();
-    expect(screen.getByText(mockBpmnDetails.name)).toBeInTheDocument();
+  it.each([
+    {
+      task: BpmnTypeEnum.Process,
+      expectedText: 'process_editor.configuration_panel_no_task_title',
+    },
+    {
+      task: BpmnTypeEnum.SequenceFlow,
+      expectedText: 'process_editor.configuration_panel_element_not_supported_message',
+    },
+    {
+      task: BpmnTypeEnum.StartEvent,
+      expectedText: 'process_editor.configuration_panel_element_not_supported_message',
+    },
+    {
+      task: BpmnTypeEnum.EndEvent,
+      expectedText: 'process_editor.configuration_panel_element_not_supported_message',
+    },
+  ])('should display correct message based on selected task', ({ task, expectedText }) => {
```

Comment:
Are these actually _tasks_ ? Isn't Task a bpmn _type_ on the same level as all these, which are then also _types_? Or? 😅

Reply:
I agree on this one, quite a bad name here. 🙈 What about `should display a correct message based on bpmnType`? Then it's consistent with the enum.